public/javascripts/gformSessionTimeout.js

```diff
@@ -0,0 +1,36 @@
+(function(global) {
+  "use strict";
+
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};
+  var lang = (global.gform && global.gform.lang) || "en";
+
+  function GformSessionTimeout() {
+    var self = this;
+
+    function init() {
+      $("input").keypress(broadcastTimestamp);
```

Comment:
Can we throttle this and broadcast only every 5 minutes or so?

Reply:
done